README.md

```diff
@@ -15,39 +15,45 @@ The bootstrap workflow uses the following process to handle incoming EDI:
 The bootstrap uses a similar process in reverse to handle outgoing EDI. It translates JSON documents to EDI and publishes them to a Stedi bucket.
 
 
-## Prerequisites & Deployment
+## Requirements
 
-1. [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) _(minimum version: 15)_
+1. Install [Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) _(minimum version: 15)_
 
-1. Clone this repo and install the necessary dependencies:
+1. Clone the bootstrap repository and install the necessary dependencies:
 
    ```bash
    git clone https://github.com/Stedi-Demos/bootstrap.git
    cd bootstrap
    npm ci
    ```
 
-1. Go to [webhook.site](https://webhook.site/) and copy the unique URL. The demo will send output to this webhook.
+1. Create a [Stedi account](https://www.stedi.com/auth/sign-up?).
 
-1. This project uses `dotenv` to manage the environmental variables required. Rename the `.env.example` file to `.env` and amend the following variables:
+ 1. [Generate an API Key](https://www.stedi.com/app/settings/api-keys). An API key is required to authenticate with Stedi APIs and deploy Stedi functions. 
 
-   - `STEDI_API_KEY`: Your Stedi API Key - used to deploy the function and internally to interact with product APIs. If you don't already have one, you can generate an [API Key here](https://www.stedi.com/app/settings/api-keys).
-   - `DESTINATION_WEBHOOK_URL`: the unique URL copied from [webhook.site](https://webhook.site/) in the previous step.
+1. Go to [webhook.site](https://webhook.site/) and copy the unique URL. The bootstrap workflow sends output to this webhook.
```

Comment:
While you're making changes in this area, it would be good to incorporate [Gareth's feedback from Slack](https://stedi-inc.slack.com/archives/C03V2DMGH8Q/p1674515010276339) as well: 
![image](https://user-images.githubusercontent.com/39536918/214369514-050666f9-b604-4794-9b3a-9f029ed3ae88.png)

Reply:
Yes! Thank you :) I have updated accordingly!